lib/runtime-c-api/src/instance.rs

```diff
@@ -10,11 +10,9 @@ use crate::{
 };
 use libc::{c_char, c_int, c_void};
 use std::{collections::HashMap, ffi::CStr, ptr, slice};
-use wasmer_runtime::{Ctx, Global, Instance, Memory, Table, Value};
-use wasmer_runtime_core::{
-    export::Export,
-    import::{ImportObject, Namespace},
-};
+use wasmer::import::{ImportObject, Namespace};
+use wasmer::wasm::{Ctx, Global, Instance, Memory, Table, Value};
+use wasmer_runtime_core::export::Export;
```

Comment:
Why export is still needed?

Reply:
This is the Instance export, the one that is executable, which is different from a module export which is not:

I actually reexported `Export` in the tests PR that just landed so we don't have to depend on runtime-core for this anymore